tools/ci_build/requirements.txt

```diff
@@ -1,5 +1,5 @@
 # packages used by transformers tool test
-
+protobuf==3.17.0
```

Comment:
Why this change is needed?

Reply:
As we sync last time, setting protobuf version will help with pipeline broke based on https://github.com/microsoft/onnxruntime/tree/snnn/p23